src/main.cpp

```diff
@@ -626,10 +627,18 @@ int main(int argc, char** argv)
     /* Start the event listener */
     drakvuf->loop(timeout);
 
+    PRINT_DEBUG("Finished DRAKVUF loop\n");
+
+    switch (drakvuf->is_interrupted())
```

Comment:
This switch needs to happen at the very end of the function.

Reply:
Or do you specifically want to skip the `terminate()` call below?

Reply:
Yes